app/components/experiment-detail.js

```diff
@@ -43,6 +44,6 @@ export default Ember.Component.extend({
             clone.save().then(() => {
                 this.get('onClone')(clone);
             });
-        }
+        },
```

Comment:
Is no trailing commas the standard? Previously in code review I've been told to use a trailing comma so that less lines are changed if code is added below. This PR has a lot of trailing commas...